typescript/cli/src/hook/read.ts

```diff
@@ -0,0 +1,54 @@
+import { ChainName, EvmHookReader } from '@hyperlane-xyz/sdk';
+import { Address, ProtocolType } from '@hyperlane-xyz/utils';
+
+import { readChainConfigsIfExists } from '../config/chain.js';
+import { getMultiProvider } from '../context.js';
+import { log, logBlue, logRed } from '../logger.js';
+import { FileFormat, writeFileAtPath } from '../utils/files.js';
+
+/**
+ * Read Hook config for a specified chain and address, logging or writing result to file.
+ */
+export async function readHookConfig({
+  chain,
+  address,
+  chainConfigPath,
+  concurrency = 20,
```

Comment:
should we be duplicating the 20 const here when it's already in the param default?